pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -187,6 +187,25 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 		if cephCluster.Spec.External.Enable {
 			logger.Warningf("external subvolume group %q deletion is not supported, delete it manually", namespacedName)
 		} else {
+			if opcontroller.IsCleanupRequired(cephFilesystemSubVolumeGroup.GetAnnotations()) {
+				logger.Infof("starting cleanup of the ceph resources for subVolumeGroup %q", namespacedName)
+				cleanupConfig := map[string]string{
+					opcontroller.CephFSSubVolumeGroupNameEnv: cephFilesystemSubVolumeGroup.Spec.Name,
+					opcontroller.CephFSVolumeNameEnv:         cephFilesystemSubVolumeGroup.Spec.FilesystemName,
+					opcontroller.CephFSNamesaceEnv:           "csi", // TODO: is it always "csi" for cephFS
```

Comment:
We create a default "csi" subvolumegroup for each cephfilesystem, but if the user creates their own subvolumegroup CR, the name will be different. And isn't this the namespace, not the svg name?
```suggestion
					opcontroller.CephFSNamespaceEnv:           namespace
```

Reply:
This is rados namespace name where the omap are stored, its always csi for csi created PVC. i suggested above to rename this variable name to avoid confusion